meson.build

```diff
@@ -125,6 +125,7 @@ test_link_args = [
   '-Wl,-z,relro',
   '-Wl,-z,defs',
   '-Wl,-z,now',
+  '-Wl,-z,ibt,-z,shstk',
```

Comment:
Is this an implicit dependency on newer compilers?  Shouldn't this only go in if supported?

Reply:
Ya, it's only used if `cc.has_link_argument` passes. For me it passes already, which was slightly unexpected:

    Compiler for C supports link arguments -Wl,-z,ibt,-z,shstk: YES

Reply:
Well the compiler stuff for CET started getting put in a while back, so not too surprising.  OK.